include/swift/SIL/SILModule.h

```diff
@@ -263,6 +263,13 @@ class SILModule {
   /// to ensure that the module is serialized only once.
   bool serialized;
 
+  /// Set if we have registered a deserialization notification handler for
+  /// stripping ownership in non transparent functions.
```

Comment:
Can you add a note saying where this is actually set (specifically, I would just reference the pass).

```
/// Set if ...
/// lowering ownership...
///
/// This is set in the pass ...
```

Just to make it easier to find.